src/index.js

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import App from './App';
+import initCornerstone from './initCornerstone.js';
 
+initCornerstone();
```

Comment:
I think it would be better to maybe have this in the App.js component constructor. That way it keeps index.js fairly lean and any app initialization and rendering logic in the app itself.